src/packages/tests/src/__tests__/__helpers__/integrationTest.ts

```diff
@@ -349,12 +326,8 @@ async function teardownScenario(state: ScenarioState) {
   // props might be missing if test errors out before they are set.
   if (state.db) {
     await Promise.resolve(
-      state.scenario.down
-        ? state.input.database.send(state.db, state.scenario.down)
-        : undefined,
+      state.input.database.afterEach?.(state.db)
```

Comment:
mssql `pool.close()` could also be called here by adding `afterEach: () =>` to the mssql implementation, but we manually close it in the `up()` so will leave as is